documentation/schema.json

```diff
@@ -976,6 +976,16 @@
             "$ref": "#/definitions/AzureBlobEgressProviderOptions"
           }
         },
+        "S3Storage": {
```

Comment:
The schema needs to be regenerated due to changes in the tooling that were brought in with the recent merge from `main`.